src/components/ebay-skeleton/components/ebay-skeleton-avatar/index.marko

```diff
@@ -0,0 +1,15 @@
+import { processHtmlAttributes } from "../../../../common/html-attributes";
+import type { AttrClass } from "marko/tags-html";
+static var ignoredAttributes = ["as", "class"];
```

Comment:
Lets use the new syntax for this
Its somethibg like
```
$ const { as, class: className, ...rest} = input
<div class=className processHtmlAttributes(...rest)>
```
Check with Luke for the right syntax.

Reply:
This will be done before my refactor, so I think it's ok to merge now and I'll do this change after a rebase.